platforms/corda-ent/charts/node-initial-registration/values.yaml

```diff
@@ -65,41 +65,48 @@ vault:
 #                    Node Configuration                     #
 #############################################################
 nodeConf:
-  # Node external address
-  # Eg. ambassadorAddress: carrier.rce.blockchainexample.com:15010
-  ambassadorAddress:
-# Node legal name
-# Eg. legalName: O=Node,OU=Node,L=London,C=GB
-legalName: 
-# Email address
-# Eg. emailAddress: dev-node@baf.com
-emailAddress: 
-
-# Name of the cenm services
-# Ex. idmanName: idman
-# Ex. networkmapName: networkmap
-cenmServices:  
-  idmanName: 
-  networkmapName:
-
-# crlCheckSoftFail defines if CRL failure is a critical error or if we can just fail softly (by logging an error) and continuing
-crlCheckSoftFail: true
-# tlsCertCrlDistPoint defines the endpoint for retrieving the CRL (Certificate Revocation List) of the Corda Network, if empty "", not used
-# Example from Corda Network UAT network: http://crl.uat.corda.network/nodetls.crl
-# Eg. tlsCertCrlDistPoint: "", empty for current usage
-tlsCertCrlDistPoint: "" 
-# tlsCertCrlIssuer defines the X500 name of the trusted CRL issuer of the Corda Network, example from the Corda Network UAT network
-# Eg. tlsCertCrlIssuer: "CN=Test TLS Signer Certificate, OU=HQ, O=HoldCo LLC, L=New York, C=US"
-tlsCertCrlIssuer: 
-
-devMode: false
+  ambassador:
+    p2pPort: 
+    external_url_suffix: 
+    # Node external address
+    # Eg. ambassadorAddress --> p2pAddress: carrier.rce.blockchainexample.com:15010
+    p2pAddress: 
+  # Node legal name
+  # Eg. legalName: O=Node,OU=Node,L=London,C=GB
+  legalName: 
+  # Email address
+  # Eg. emailAddress: dev-node@baf.com
+  emailAddress: 
+  # crlCheckSoftFail defines if CRL failure is a critical error or if we can just fail softly (by logging an error) and continuing
+  crlCheckSoftFail: true
+  # tlsCertCrlDistPoint defines the endpoint for retrieving the CRL (Certificate Revocation List) of the Corda Network, if empty "", not used
+  # Example from Corda Network UAT network: http://crl.uat.corda.network/nodetls.crl
+  # Eg. tlsCertCrlDistPoint: "", empty for current usage
+  tlsCertCrlDistPoint: "" 
+  # tlsCertCrlIssuer defines the X500 name of the trusted CRL issuer of the Corda Network, example from the Corda Network UAT network
+  # Eg. tlsCertCrlIssuer: "CN=Test TLS Signer Certificate, OU=HQ, O=HoldCo LLC, L=New York, C=US"
+  tlsCertCrlIssuer: 
+  devMode: false
+  # Provide volume related specifications
+  volume:
+    # Ex baseDir: /opt/corda
+    baseDir:
+  # where is node jar is stored
+  jarPath: bin
+  # where service configuration files are stored
+  configPath: etc
+  cordaJarMx:

```

Comment:
Add comments from platforms/corda-ent/charts/notary-initial-registration/values.yaml

Reply:
As well as the correct memory parameters

Reply:
done changes